docs/README.adoc

```diff
@@ -343,6 +343,20 @@ openstack overcloud deploy \
 [NOTE:] 
 The SSL certificates for the `MetricsQdr` service is configured to generate only for the `InternalApi` network but the default Ceph role does not use the `InternalApi` network. To deploy SAF client when InternalTLS is enabled, use this workaround: pass the custom Ceph role, which has `InternalApi` network, to `openstack overcloud deploy` when InternalTLS is enabled in the deployment.
 
+[id=`configuration-for-distributed-compute-nodes_{context}`]
+=== Configuration for distributed compute nodes
+If you are deploying with compute nodes on different networks to the controller, some additional changes are need so that the metrics sent over AMQ can reach the SAF.
+The following needs to be added under the `ExtraConfig` section for each compute node.
+
+----
+ComputeNExtraConfig:
+	tripleo::profile::base::metrics::collectd::amqp_host: "%{hiera('internal_apiN')}"
+	tripleo::profile::base::metrics::qdr::listener_addr: "%{hiera('internal_apiN')}"
+----
+
+In the above example, `internal_apiN` is the network that the compute node is connected to.
```

Comment:
I'd place this above the example so that when someone is looking at it they understand what items to call out in their head. So I'd reword it a bit like so:

The following configuration example needs to be added under the `ExtraConfig` section for each compute node.

In the example, `internal_apiN` is the network that the compute node is connected to, where `N` represents a value different from the default internal API network name.

```
[source,yaml]
----
code block
----
```

For more information on this configuration, see: https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/13/html-single/spine_leaf_networking/